variables.tf

```diff
@@ -177,3 +177,34 @@ variable "routing_hosts" {
   default     = []
   description = "List of host header patterns to match and the target_group_arn to forward traffic to."
 }
+
+
```

Comment:
More details on the CORS Configuration variables [here](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html#how-do-i-enable-cors)